src/graphql/middleware/responseTransactionIdMiddleware.ts

```diff
@@ -0,0 +1,22 @@
+import { ServerError } from "../../utils/errorHelper";
+import { MiddlewareFn } from "type-graphql";
+import uuid4 from "uuid4";
+
+export const ResponseTransactionIdMiddleware: MiddlewareFn<any> = async (
+	{ context },
+	next
+) => {
+	if (context.responseTranscationId) {
+		return next();
+	}
+
+	context.responseTranscationId = true;
+	try {
+		await next();
+		const txId = uuid4();
```

Comment:
rename variable txId to transactionId

Reply:
This is being renamed as correlationId in the recent commits.